react.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+module.exports = {
+  "rules": {
+
+    /**
+     * React style
+     */
+    "react/display-name": 0,              // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md
```

Comment:
I'm guessing that for react, that the `0`, `1`, `2` values lines up with the regular ESLint values, but what do you think about changing these to `"off"`, `"warn"`, `"error"`?

I changed the values in the default one because I think it's easier to see `"error"` at a glance and know what it means rather than `2` and remembering that `2` means `"error"`. Regardless, I think we should be consistent.